### PR TITLE
DetailSite

### PR DESCRIPTION
**Pull request by jk0ehn**

### CODE REVIEW

File: src/components/Cards/ExpertList.js [hunk 1]
```diff
@@ -8,6 +9,11 @@ const Ul = styled.ul`
 	list-style: none;
 `;
 
+const Anchor = styled.a`
+	color: black;
+	text-decoration: none !important;
```
Comment (julialoeschel) [inline src/components/Cards/ExpertList.js]: are you sure there is no other way to style this link correctly? 
we try not to use `important`if we can find a better solution ;)

File: src/components/Cards/ExpertList.js [hunk 2]
```diff
@@ -36,43 +42,47 @@ const City = styled.div`
...
-								<div>
-									<text>
-										{item.first_name} {item.last_name}
-									</text>
-									<br />
-									<text>{item.ident}</text>
-								</div>
+									<DetailPreview>
+										<div>
+											<text>
```
Comment (julialoeschel) [inline src/components/Cards/ExpertList.js]: I never saw this html element before - where did you found this? and why are you not using a `<p> `tag, or a `<span>` - which would be semantically correct?

File: src/components/Cards/ExpertList.js [hunk 3]
```diff
@@ -36,43 +42,47 @@ const City = styled.div`
...
+												{item.first_name} {item.last_name}
+											</text>
+											<br />
```
Comment (julialoeschel) [inline src/components/Cards/ExpertList.js]: Also - we are not using `<br>` for styling. 
please style your application with css - not with html :)

File: src/components/Cards/ExpertList.js [hunk 4]
```diff
@@ -36,43 +42,47 @@ const City = styled.div`
...
+										</div>
 
-								<div>
-									<City>{item.city}</City>
-								</div>
-							</DetailPreview>
-						</Card>
+										<div>
+											<City>{item.city}</City>
+										</div>
```
Comment (julialoeschel) [inline src/components/Cards/ExpertList.js]: are you sure you need that div here?

File: src/components/Footer.js [hunk 5]
```diff
@@ -1,7 +1,22 @@
...
+	font-size: 24px;
+`;
+
 export default function Footer() {
 	return (
-		<footer>
-			<div>© 2022 by Me</div>
-		</footer>
+		<GreenFooter>
+			<div>Find your local expert!</div>
```
Comment (julialoeschel) [inline src/components/Footer.js]: use a `<p>`tag or a `<span>`for text elements. 
Also you should try to use as less divs as possible. Your Styled component should be a `<p>`, so the extra div is not necessary.

File: src/pages/experts/[ident].js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+							objectFit="cover"
+							alt="ProfilePic"
+						/>
+					</Img>
+
+					<TextArea>
+						<div>ID: {expert.ident}</div>
+						<div>{expert.city}</div>
+						<div>{expert.state}</div>
+					</TextArea>
```
Comment (julialoeschel) [inline src/pages/experts/[ident].js]: this is incorrect html. please check `https://caninclude.glitch.me/` and read about the usage of `<p>` 🚁

**Review by julialoeschel — COMMENTED** [review]
Hey, I found some critical HTML issues here. 
I suggest take an extra hour to deepdive into html Elements and semantic html.

Cool thougth, that you made the svgs work. 
I am exictest how they look styled ! 

good you Julian

File: src/pages/experts/[ident].js (hunk 6 above)
Comment (jk0ehn) [reply src/pages/experts/[ident].js]: How should I use this link?

File: src/components/Cards/ExpertList.js (hunk 1 above)
Comment (jk0ehn) [reply src/components/Cards/ExpertList.js]: it works without !import !? Any other advices?ideas?

File: src/components/Header.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+	color: white;
+	font-size: 24px;
+`;
+
+export default function Header(props) {
+	return (
+		<GreenHeader>
+			<div>
+				{props.first_name} {props.last_name}
+			</div>
```
Comment (julialoeschel) [inline src/components/Header.js]:
```suggestion
			
				{props.first_name} {props.last_name}
			
```

File: src/components/Header.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+
+const GreenHeader = styled.div`
```
Comment (julialoeschel) [inline src/components/Header.js]:
```suggestion
const GreenHeader = styled.span`
```

**Review by julialoeschel — COMMENTED** [review]
good job! 
I still found some unnecessary divs  👀 

but thats all - great ! 
you can merge it yourself after handling the last issue
;)

**Pull request merged**